tvb_documentation/manuals/UserGuide/UserGuide-UI_Connectivity.rst

```diff
@@ -502,3 +503,53 @@ It will produce a Structural Connectivity in TVB format and a compatible brain V
 Check the Project --> Operations page to see when the import from Allen is done.
 You can also find your resulted Connectivity in Project --> Data Structure area.
 
+
+Siibra Connectivity Creator
+...........................
+In this page you can use the `siibra library <https://github.com/FZJ-INM1-BDA/siibra-python>`_
+to create structural and functional connectivities using data from
+to the `EBRAINS Knowledge Graph <https://kg.ebrains.eu/>`_. This knowledge graph
+brings together information from different data sources regarding
+brain atlases, parcellations and their associated features for multiple species. To be able to access
+this information, you will need a special token provided by the EBRAINS team.
+For more information, please refer to this `page <https://kg.humanbrainproject.eu/develop.html>`_.
+
+Currently, the structural connectivities are stored as TVB Connectivities and the
+functional connectivities are stored as TVB Connectivity Measures.
+
+    .. figure:: screenshots/siibra_connectivity.png
```

Comment:
Could this be new screenshot be made `.JPG` as the rest of the screenshots, for keeping them consistent ?